firefox_puppeteer/__init__.py

```diff
@@ -47,6 +47,14 @@ def keys(self):
         See the :class:`~api.keys.Keys` reference.
         """
 
+    @use_class_as_property('api.permissions.Permissions')
+    def perms(self):
+        """
+        Provides an api for interacting with permissions.
+
+        See the :class:`~api.permissions.Permissions` reference.
```

Comment:
You will also have to add the appropriate documents under docs/api for the new library. Let me know which PR you want to use for that implementation so we don't have to share that information between the two for safebrowsing.